punchcut/src/source/glyf/outline.rs

```diff
@@ -0,0 +1,165 @@
+use super::Point;
+use crate::PathSink;
+
+/// TrueType outline.
+#[derive(Default, PartialEq, Eq, Debug)]
+pub struct Outline {
+    /// Set of points that define the shape of the outline.
+    pub points: Vec<Point>,
+    /// Set of tags (one per point).
+    pub tags: Vec<u8>,
```

Comment:
Why not Vec(Point, PointTag) then? - would make the tag-per-point relation explicit and unscrewupable.

Reply:
A lot of context is lost with hinting and variations being missing. The tags array is also used during delta computation and hinting and is temporarily associated with other point arrays (original, unscaled, deltas) during those processes.